src/css/custom.scss

```
@@ -1,12 +1,31 @@
+@use "./admonitions";
+@use "./details";
+@use "./tabs";
+@use "./code-blocks";
+@use "./tables";
+@use "./feedback";
+@use "./back-to-top";
+@use "./edit-this-page";
+
+// Fonts
+
+@import url("https://fonts.googleapis.com/css2?family=Funnel+Display:wght@300..800&family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap");
```

Stylelint (17.14.1)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Funnel+Display:wght@300..800&family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Funnel+Display:wght@300..800&family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap"" (import-notation)

(import-notation)